src/helpers/recruiter/api.ts

```diff
@@ -0,0 +1,474 @@
+  company?: {
+    name?: string;
+    website?: string;
+    domains?: any[];
+    category?: string;
+    address?: {
+      line1: string;
+      line2: string;
+      city: string;
+      state: string;
+      country: string;
+    };
+    size?: number;
+    yearOfEstablishment?: string;
+    annualTurnover?: string;
+    socialMediaLink?: string;
+  };
+  user?: {
+    name: string;
+    email: string;
+    contact: string;
+  };
+}
+
+export const patchProfile = async (
+  accessToken: string | undefined,
+  changes: updateProfileFC
+) => {
+  if (!accessToken || accessToken === undefined) {
+    redirect();
+    return;
+  }
+  const res = await fetch(url(`/recruiter-view/recruiter`), {
+    method: "PATCH",
+    headers: {
+      "Content-type": "application/json",
+      Authorization: `Bearer ${accessToken}`,
+    },
+    body: JSON.stringify(changes),
+  });
+  return res.ok;
+};
+
+export interface JobsFC {
+  id: string;
+  role: string;
+  active: boolean;
+  currentStatus: string;
+  season: {
+    id: string;
+    year: string;
+    type: string;
+  };
+  company: {
+    id: string;
+    name: string;
+  };
+}
+
+export const getJobs = async (accessToken: string | undefined) => {
+  if (!accessToken || accessToken === undefined) {
+    redirect();
+    return;
+  }
+  const res = await fetch(url(`/recruiter-view/jobs`), {
+    cache: "no-store",
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+  const json: JobsFC[] = await res.json();
+  return json;
+};
+
+export interface JobDetailFC {
+  id: string;
+  role: string;
+  active: boolean;
+  currentStatus: string;
+  season: {
+    id: string;
+    year: string;
+    type: string;
+  };
+  company: {
+    id: string;
+    name: string;
+  };
+  selectionProcedure: {
+    selectionMode: string;
+    shortlistFromResume: true;
+    groupDiscussion: true;
+    tests: [
+      {
+        type: string;
+        duration: number;
+      },
+    ];
+    interviews: [
+      {
+        type: string;
+        duration: number;
+      },
+    ];
+    requirements: {
+      numberOfMembers: number;
+      numberOfRooms: number;
+      otherRequirements: string;
+    };
+    others: string;
+  };
+  description: string;
+  attachment: string;
+  skills: string;
+  offerLetterReleaseDate: string;
+  joiningDate: string;
+  location: string;
+  noOfVacancies: number;
+  duration: number;
+  feedback: string;
+  jobCoordinators: [
+    {
+      id: string;
+      role: string;
+      tpcMember: {
+        id: string;
+        department: string;
+        role: string;
+        user: {
+          id: string;
+          email: string;
+          name: string;
+          contact: string;
+        };
+      };
+    },
+  ];
+  events: [
+    {
+      id: string;
+      roundNumber: number;
+      type: string;
+      metadata: string;
+      startDateTime: string;
+      endDateTime: string;
+      visibleToRecruiter: true;
+    },
+  ];
+  salaries: [SalaryFC];
+}
```

Comment:
make a separate file in helpers/recruiter named types.ts and store all the interfaces there